templates/debug.hbs

```diff
@@ -1,19 +1,45 @@
 <style>
-    svg:first-child {
-        display: none;
+    .u-hidden{
+        display:none;
     }
-    svg {
+
+    code{
+        font-familly:monospace;
+        font-size:11px;
+        color:#666;
+        display:block;
+    }
+
+    .wrap{
+        display: flex;
+        flex-wrap: wrap;
+    }
+
+    div.item{
```

Comment:
In general I'd like to have spaces between the selector and the curly brace, as well as between the colon and value in properties, and between selectors.
```
selector selector > selector {
   property: value;
}

Reply:
i agree